fms2_io/netcdf_io.F90

```diff
@@ -245,9 +246,24 @@ module netcdf_io_mod
 contains
 
 !> @brief Accepts the namelist fms2_io_nml variables relevant to netcdf_io_mod
-subroutine netcdf_io_init (chksz)
+subroutine netcdf_io_init (chksz, netcdf_default_format)
 integer, intent(in) :: chksz
+character (len = 10), intent(in) :: netcdf_default_format
+
  fms2_ncchksz = chksz
+ if (string_compare(netcdf_default_format, "64bit", .true.)) then
+     fms2_nc_format_param = nf90_64bit_offset
+     call string_copy(fms2_nc_format, "64bit")
+ elseif (string_compare(netcdf_default_format, "classic", .true.)) then
+     fms2_nc_format_param = nf90_classic_model
+     call string_copy(fms2_nc_format, "classic")
+ elseif (string_compare(netcdf_default_format, "netcdf4", .true.)) then
+     fms2_nc_format_param = nf90_netcdf4
+     call string_copy(fms2_nc_format, "netcdf4")
+ else
+     call error("unrecognized netcdf file format "//trim(netcdf_default_format)//". Check fms2io namelist")
```

Comment:
Can you list the possible netcdf file formats.  A user will not know what to do with this error.  The list isn't that long.